docs/clients/topic-client.md

```
@@ -0,0 +1,279 @@
+topicClient.deleteTopic(topicId);
+```
+
+```java title="deleteTopic(TopicId topicId, PrivateKey adminKey)"
+TopicId topicId =
+    TopicId.fromString("0.0.1234");
+
+topicClient.deleteTopic(
+    topicId,
+    adminKey
+);
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)